lib/commandCenter/commandCenter.js

```diff
@@ -788,9 +789,18 @@ CommandCenter.prototype.executeBatch = function (acceptedEncodings, batch, query
 
 			// recursively runs each command, until done, then calls respond
 
-			function runCommand(output, cmdIndex) {
+			function runCommand(output, cache, cmdIndex) {
 				var cmd = batch.commands[cmdIndex];
 				if (cmd) {
+					if (cacheContent.length !== 0 && isCachedCommand(cmd)) {
+						logger.warning
+							.data({ cmd: cmd.cmdName })
+							.log('Re-sending cached command response');
+
+						output += (output ? ',' : '[') + cacheContent.shift();
+						return setImmediate(runCommand, output, cache, cmdIndex + 1);
```

Comment:
The signature here and on line 823 seem to disagree with each other.

Reply:
Fixed the signature.